### DISCUSSION TITLE
Visibility Problem on Windows

### DISCUSSION THREAD

**Discussion by DrownedStarfish** [standardnotes/forum#2473] [General] [answered]
Hi!

I'm running SN v. 3.5.11 on Win7 Pro SP1 64bits (aside from running SN on some non-Windows devices).

On Windows, when I go into any of the three panes that I see when I have a note open, the vertical scroll bar on the right is too light-colored to be easily visible and too narrow to be easily maneuvrable.

Thanks.
D.

**Comment by JaspalSuri** [comment]
Hi @DrownedStarfish,

We're aware of the issue where you can't drag the scrollbar and we plan to address this in future releases of the web & desktop apps.

Does the scrollbar look like this on your end?
![image](https://user-images.githubusercontent.com/44656000/103356322-77621100-4a65-11eb-9364-4be22484933b.png)

**Comment by DrownedStarfish** [comment]
Mine is a much lighter gray, which is more difficult to see.

Thanks.
D.

**Comment by myreli** [comment, marked as answer]
We are triaging older issues and it seems like this is no longer a problem with the recent UI updates. 

Let us know if it happens again. Thank you!
